src/tests/suite/SConscript

```diff
@@ -105,13 +105,13 @@ def scons():
                     daos_aggregate_ec.c daos_degrade_ec.c
                     daos_extend_simple.c daos_obj_ec.c""")
 
-    newenv.AppendUnique(LIBPATH=["$BUILD_DIR/src/client/dfs"])
+    newenv.AppendUnique(LIBPATH=[Dir('../../client/dfs')])
     daostest = daos_build.program(newenv, 'daos_test', c_files + daos_test_tgt,
                                   LIBS=['daos_common'] + libraries)
 
     c_files = Split("""dfs_unit_test.c dfs_par_test.c dfs_test.c """
                     """dfs_sys_unit_test.c""")
-    newenv.AppendUnique(CPPPATH=["#/src/client/dfs"])
+    newenv.AppendUnique(CPPPATH=[Dir('../../client/dfs').srcnode()])
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string